CHANGELOG.md

```diff
@@ -1,5 +1,616 @@
 # Changelog
 
+## 12.0.0
+
+Teleport 12 brings the following marquee features and improvements:
+
+- Device Trust (Preview, Enterprise only)
+- Passwordless Windows access for local users (Preview, Enterprise only)
+- Per-pod RBAC for Kubernetes Access
+- Azure and GCP CLI support for Application Access
+- Support for more databases in Database Access
+- AWS DynamoDB
+- AWS Redshift Serverless
+- AWS RDS Proxy for PostgreSQL/MySQL
+- Azure SQLServer Auto Discovery
+- Azure Flexible Servers
```

Comment:
Should these be indented bullets under the previous point?

Reply:
Nope, this is a separate "feature".

Reply:
Actually, I misunderstood your comment. Yes, they should be, I'll update.